example/App.js

```diff
@@ -319,25 +319,31 @@ export const App = () => {
                 title="toggleMinMaxDate"
               />
             </View>
-            {show && (
-              <DateTimePicker
-                testID="dateTimePicker"
-                timeZoneOffsetInMinutes={tzOffsetInMinutes}
-                minuteInterval={interval}
-                maximumDate={maximumDate}
-                minimumDate={minimumDate}
-                value={date}
-                mode={mode}
-                is24Hour
-                display={display}
-                onChange={onChange}
-                style={styles.iOsPicker}
-                textColor={textColor || undefined}
-                accentColor={accentColor || undefined}
-                neutralButtonLabel={neutralButtonLabel}
-                disabled={disabled}
-              />
-            )}
+            <View style={{flexDirection: 'row', alignItems: 'center'}}>
```

Comment:
This should be removed before merging – it serves only as a showcase of the issue.

Reply:
actually, I think we can even keep it. You can reference the issue #409 in the label so that everyone know why it's there :)

Reply:
Ok, I will keep it then 👍